src/mongoose_client_api/mongoose_client_api_messages.erl

```diff
@@ -142,10 +149,12 @@ encode(Msg, Timestamp) ->
          {<<"to">>, exml_query:attr(Msg, <<"to">>)},
          {<<"id">>, exml_query:attr(Msg, <<"id">>)},
          {<<"body">>, exml_query:cdata(BodyTag)},
-         {<<"timestamp">>, Timestamp} | ExtensionList],
-
+         {<<"thread">>, Thread},
+         {<<"timestamp">>, Timestamp}],
+    J = lists:append(L, ExtensionList),
```

Comment:
L stands for list, J and K names are a bit confusing. Can we use L1, L2 names instead?

Reply:
Okay. Changing it now.